controls/srg_gpos/SRG-OS-000358-GPOS-00145.yml

```diff
@@ -1,10 +1,21 @@
 controls:
     -   id: SRG-OS-000358-GPOS-00145
         levels:
-            - medium
+            - low
```

Comment:
is there a specific reason for this change?

Reply:
To match the RHEL8 STIG.

Reply:
But the SRG states this as medium

```
severity="medium"><version>SRG-OS-000358-GPOS-00145</version>
```

I think there are some inconsistencies in the RHEL8 STIG initial spreadsheet.

https://cyber.trackr.live/stig/General_Purpose_Operating_System/2/2

It should be safe to revert IMO